src/cmdutil.ts

```diff
@@ -1,5 +1,5 @@
-const directline = require("../dist/bridge.js");
```

Comment:
`.ts` file from source shouldn't depend on anything from build output.

Reply:
good point. Are you using the cmd line util? this is actually largely vestigial

Reply:
> Are you using the cmd line util?
No.  Just thought I would fix it while I was here although I don't think it's necessary for this PR and could be reverted.  I'm not sure about the history of this repo, but it seems like it is being used by more people and for more use cases than originally intended so I wanted to make it more robust where I could.

For example, if another author unfamiliar clones this and changes the `tsconfig` to output the compiled files to a `build` folder instead of `dist` which should have no affect on the library it would break this file unintentionally.